src/ucp/rma/rma.inl

```diff
@@ -95,4 +94,30 @@ static inline void ucp_ep_rma_remote_request_completed(ucp_ep_t *ep)
     }
 }
 
+static inline ucs_status_t ucp_rma_sw_do_am_bcopy(ucp_request_t *req, uint8_t id,
+                                                  uct_pack_callback_t pack_cb,
+                                                  ssize_t *packed_len_p)
```

Comment:
no need to return packed_len

Reply:
we need the `packed_len` in the `ucp_rma_sw_progress_put()`

Reply:
ok